modules/microshift-default-settings.adoc

```diff
@@ -39,39 +39,41 @@ etcd:
-  status: Managed # <7>
-manifests: # <8>
+    namespaceOwnership: InterNamespaceAllowed # <8>
+  status: Managed # <9>
+manifests: # <10>
   kustomizePaths:
     - /usr/lib/microshift/manifests
     - /usr/lib/microshift/manifests.d/*
     - /etc/microshift/manifests
     - /etc/microshift/manifests.d/*
 network:
   clusterNetwork:
-    - 10.42.0.0/16 # <9>
+    - 10.42.0.0/16 # <11>
   serviceNetwork:
-    - 10.43.0.0/16 # <10>
-  serviceNodePortRange: 30000-32767 # <11>
+    - 10.43.0.0/16 # <12>
+  serviceNodePortRange: 30000-32767 # <13>
 node:
-  hostnameOverride: "" # <12>
-  nodeIP: "" # <13>
+  hostnameOverride: "" # <14>
+  nodeIP: "" # <15>
 ----
 <1> A string that specifies the IP address from which the API server is advertised to members of the cluster. The default value is calculated based on the address of the service network.
 <2> Subject Alternative Names for API server certificates.
 <3> Log verbosity. Valid values for this field are `Normal`, `Debug`, `Trace`, or `TraceAll`.
 <4> By default, `etcd` uses as much memory as needed to handle the load on the system. However, in memory constrained systems, it might be preferred or necessary to limit the amount of memory `etcd` can to use at a given time.
 <5> Base domain of the cluster. All managed DNS records are subdomains of this base.
-<6> Describes how host name claims across namespaces are handled. By default, allows routes to claim different paths of the same host name across namespaces. Can also be set as `Strict` to not allow routes in different namespaces to claim the same host. If the value is deleted and a new one is not added, `InterNamespaceAllowed` is automatically set.
-<7> Default router status, can be `Managed` or `Removed`.
-<8> The locations on the filesystem to scan for `kustomization` files to use to load manifests. Set to a list of paths to scan only those paths. Set to an empty list to disable loading manifests. The entries in the list can be glob patterns to match multiple subdirectories.
-<9> A block of IP addresses from which Pod IP addresses are allocated. This field is immutable after installation.
-<10> A block of virtual IP addresses for Kubernetes services. IP address pool for services. A single entry is supported. This field is immutable after installation.
-<11> The port range allowed for Kubernetes services of type NodePort. If not specified, the default range of 30000-32767 is used. Services without a `NodePort` specified are automatically allocated one from this range. This parameter can be updated after the cluster is installed.
-<12> The name of the node. The default value is the hostname. If non-empty, this string is used to identify the node instead of the hostname.
-<13> The IP address of the node. The default value is the IP address of the default route.
+<6> The `ingress.listenAddress` value defaults to the entire network of the host. Valid customizable values can be a single IP address or host name or a list of IP addresses or host names.
```

Comment:
At the end of the sentence it should mention NIC names, not host names. Also, not sure whether we should just mention it is a list (which allows 1 or more values)?

Reply:
example of specifying the host's ip address
```
ingress:
  listenAddress:
    - 10.2.1.100
```
 
example of specifying a list of the host's ip address(or NIC name)
```
ingress:
  listenAddress:
    - 10.2.1.100
    - 10.2.2.10
    - ens3
```

Reply:
Updated the default value footnote and added these example configs to the `microshift-nw-router-config-IPs.adoc` module.